ABDataCollectionCore.js

```diff
@@ -2440,12 +2443,8 @@ module.exports = class ABDataCollectionCore extends ABMLClass {
             }
 
             // now we close out our .loadData() promise.resolve() :
-            if (this._pendingLoadDataResolve) {
-               this._pendingLoadDataResolve.resolve();
-
-               // after we call .resolve() stop tracking this:
-               this._pendingLoadDataResolve = null;
-            }
+            this._pendingLoadDataResolves[data.jobID].resolve();
```

Comment:
let's make sure we account for `data.jobID` that isn't there, just in case.  
```
if (data.jobID) {
  this._pendingLoadDataResolves( ...)
  delete ...
}
```